spec/features/admin/projects_spec.rb

```diff
@@ -4,7 +4,8 @@
 
 describe 'Projects', type: :feature do
   let(:admin_user) { create(:user, :admin, occupation: :administrative) }
-  let!(:project)   { create(:project, company: admin_user.company) }
+  let!(:client) { create(:client, company: admin_user.company) }
```

Comment:
This exclamation mark seems to be unnecessary here since `client` is called right below in `project`